src/commands.py

```diff
@@ -478,25 +469,10 @@ def _get_user_for_reply(message: discord.Message, message_forwarder: MessageForw
     # If it's a reply thread, the user the reply thread is for, otherwise None
```

Comment:
Github astoundingly doesn't allow commenting on arbitrary lines in the PR, only 'near' ones that changed.

Unrelated to this line - this function doesn't need the `am` argument anymore, can remove it.